### ISSUE TITLE
RuntimeError: istft requires a complex-valued input tensor matching the output from stft with return_complex=True.

### ISSUE THREAD

**Issue by nileshchilka1** [espnet/espnet#5227]
RuntimeError: istft requires a complex-valued input tensor matching the output from stft with return_complex=True.

**Comment by sw005320** [comment]
@Emrys365, can you confirm it?
Is it included in your PR in https://github.com/espnet/espnet/pull/5193?

**Comment by Emrys365** [comment]
Yes, this is a known issue in PyTorch 2.0+ (https://github.com/espnet/espnet/issues/5203), which has not been officially supported yet (https://github.com/espnet/espnet/pull/5117).
